src/index.html

```diff
@@ -18,5 +18,66 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="aimg"
+        href="dfrf"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          class="nav__link is_active"
+          href="1"
```

Comment:
Don't use numbers for `<a>` tag's `href` property. It's better to use meaningful identifiers or paths.